### PR TITLE
45979189unet3d pull request

### PR DESCRIPTION
**Pull request by aCoalBall**
Hi there,

It's a pull request from Dawei Shen.

I added a folder 45979189unet3d, which contains a working project of task4 in the report's task sheet.

The folder contains a model class, model.py and a script for training and testing, driver.py. It also contains some figures and graphs for illustrating my progress.

cheers,
Dawei Shen

### CODE REVIEW

**Comment by wooboyeong** [comment]
Code
* Poor segmentation performance
* Did you normalise the images?
* Batch normalisation may also help
* Might need to train for longer

README
* When showing example outputs, also show the ground truth mask

General Comments
* Commit messages OK mostly but could be more descriptive
* Pull request OK

**Comment by shakes76** [comment]
## TF/Torch Usage
Adequate use and implementation, poor performance -2
## Good Practice (Design/Commenting)
Good spacing and comments
Header blocks
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present 
No Data leakage 
Difficulty: Normal -5
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch)
Please remove model files, these will clog up the repository. Your pull request will not be merged until they are removed. Feedback marks will be awarded when these are removed. -2
Request Description could be more informative -1

**Comment by shakes76** [comment]
Feedback marks applied
